### DISCUSSION TITLE
Changing Battery Options

### DISCUSSION THREAD

**Discussion by youngy00** [egguy/grott-home-assistant-add-on#42] [General]
Hi,

First of all, great work on this project.

I am looking into my automations and was wondering if we can change the charging values (Battery First) via Grott? I want to be able to automate the charging of my battery depending on the time of day (due to time of day electricity prices) I currently change manually on the Growatt website but would like to automate this deepening certain conditions.

![image](https://github.com/egguy/grott-home-assistant-add-on/assets/55174793/79edc107-ed02-43b5-97a2-a73acebb0be2)

Cheers,

**Comment by KasperHolchKragelund** [comment]
Hi,
I had the exact same issue and developed this to control battery charging, export limit and grid export time:
https://github.com/KasperHolchKragelund/ad-growatt
(based on the original code: https://github.com/mjdyson/ad-growatt)

Cheers
